extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -213,35 +230,42 @@ func CreateRKE2K3SSnapshot(client *rancher.Client, clusterName string) error {
 }
 
 // RestoreRKE1Snapshot is a helper function to restore a snapshot on an RKE1 cluster. Returns error if any.
-func RestoreRKE1Snapshot(client *rancher.Client, clusterName string, snapshotRestore *management.RestoreFromEtcdBackupInput, initialControlPlaneValue, initialWorkerValue string) error {
-	clusterID, err := clusters.GetClusterIDByName(client, clusterName)
+func RestoreRKE1Snapshot(client *rancher.Client, cluster *management.Cluster, snapshotRestore *management.RestoreFromEtcdBackupInput) error {
+	clusterID, err := clusters.GetClusterIDByName(client, cluster.Name)
```

Comment:
everything after this is using the clusterID, do we really need the entire cluster object here?

Reply:
I'll update to just pull the cluster name.

Reply:
Resolved.